config/initializers/sidekiq.rb

```diff
@@ -6,6 +6,7 @@
 end
```

Comment:
Shall we add: https://github.com/alphagov/link-checker-api/pull/616/files#diff-fe63ad82c410f651197078bd21118c1fa7a9dbb0a468e90940e76ba6c07e87eaR22-R24 to prevent config regressions?